Gopkg.toml

```diff
@@ -22,6 +22,15 @@ required = [
   unused-packages = true
   non-go = true
 
+[[prune.project]]
+  name = "k8s.io/code-generator"
+  non-go = false
+  unused-packages = false
+
+[[prune.project]]
+  name = "k8s.io/gengo"
+  unused-packages = false 
```

Comment:
Looks the added files from gengo are all from examples directory, are we using them?

Reply:
Yeah we actually need this package k8s.io/gengo/examples/defaulter-gen/generators